backend/infrahub/core/schema/definitions/core/action.py

```diff
@@ -0,0 +1,260 @@
+    order_by=["name__value"],
+    display_labels=["name__value"],
+    branch=BranchSupportType.AGNOSTIC,
+    uniqueness_constraints=[["name__value"]],
+    generate_profile=False,
+    inherit_from=[InfrahubKind.ACTION],
+    relationships=[
+        Rel(
+            name="generator",
+            peer=InfrahubKind.GENERATORDEFINITION,
+            kind=RelKind.ATTRIBUTE,
+            cardinality=Cardinality.ONE,
+            identifier="core_generator_action__generator_definition",
+            optional=False,
+            order_weight=4000,
+        ),
+    ],
+)
+
+core_group_action = NodeSchema(
+    name="GroupAction",
+    namespace="Core",
+    include_in_menu=True,
+    icon="mdi:cards-diamond-outline",
+    label="Group Action",
+    default_filter="name__value",
+    order_by=["name__value"],
+    display_labels=["name__value"],
+    branch=BranchSupportType.AGNOSTIC,
+    uniqueness_constraints=[["name__value"]],
+    generate_profile=False,
+    inherit_from=[InfrahubKind.ACTION],
+    attributes=[
+        Attr(
+            name="action_type",
+            kind="Text",
+            enum=GroupAction.available_types(),
+            optional=False,
+            order_weight=3000,
+        ),
+    ],
+    relationships=[
+        Rel(
+            name="group",
+            peer=InfrahubKind.GENERICGROUP,
+            kind=RelKind.ATTRIBUTE,
+            cardinality=Cardinality.ONE,
+            identifier="core_action__group",
+            optional=False,
+            order_weight=4000,
+        ),
+    ],
+)
+
+
+core_node_trigger_rule = NodeSchema(
+    name="NodeTriggerRule",
+    namespace="Core",
+    include_in_menu=True,
+    icon="mdi:filter-cog-outline",
+    label="Node Trigger Rule",
+    default_filter="name__value",
+    order_by=["name__value"],
+    display_labels=["name__value"],
+    branch=BranchSupportType.AGNOSTIC,
+    uniqueness_constraints=[["name__value"]],
+    generate_profile=False,
+    inherit_from=[InfrahubKind.TRIGGERRULE],
+    attributes=[
+        Attr(
+            name="node_kind",
+            kind="Text",
+            unique=False,
+            optional=False,
+            order_weight=3000,
+        ),
+        Attr(
+            name="mutation_action",
+            kind="Text",
+            enum=NodeAction.available_types(),
+            unique=False,
+            optional=False,
+            order_weight=4000,
+        ),
+        Attr(
+            name="schema_path",
+            kind="Text",
+            unique=False,
+            optional=False,
+            order_weight=5000,
+        ),
+        Attr(
+            name="value",
+            kind="Text",
+            unique=False,
+            optional=False,
+            order_weight=6000,
+        ),
+        Attr(
+            name="value_updated_to",
+            kind="Boolean",
+            unique=False,
+            optional=False,
+            default_value=True,
+            order_weight=7000,
+        ),
+        Attr(
```

Comment:
Would be good to align that with what we are doing for webhook (`branch_scope`)

Reply:
Switched to the same kind of setup.